alacritty/src/display.rs

```diff
@@ -275,12 +275,12 @@ impl Display {
         }
 
         #[allow(clippy::single_match)]
+        #[cfg(not(windows))]
```

Comment:
I'm honestly not sure if putting this out here isn't more complicated than it has to be. Performance difference shouldn't matter at all, since it'll get optimized out, but having everything in the same place makes sense to me. So I'd propose reverting this extraction.

Reply:
it was to fix compilation error, since the match was empty.

Reply:
Error, or warning?

Reply:
yes.